app/views/people/show.html.erb

```diff
@@ -16,24 +13,16 @@
         {
           text: "Biography",
           href: "#biography"
-        }
-      ]
+        },
+        person.has_previous_roles? ? {
```

Comment:
The contents heading and links shouldn't be part of the the image card component. It means that 'Contents' isn't a heading and on smaller screens it sits next to the image rather than below it.

Before:
<img width="282" alt="Screenshot 2019-11-15 at 14 57 19" src="https://user-images.githubusercontent.com/1732331/68952722-47e83a80-07b8-11ea-80ac-7873b06c2cd7.png">

After:
<img width="307" alt="Screenshot 2019-11-15 at 14 57 09" src="https://user-images.githubusercontent.com/1732331/68952727-4cacee80-07b8-11ea-82c8-e05ea4bab30a.png">

Reply:
Oh interesting! Is there an alternative component available or do we construct that from ordinary HTML?

Reply:
Hmm 🤔 the [contents list](https://components.publishing.service.gov.uk/component-guide/contents_list) component would be the best - there's a visual difference with the heading not being bolded, but it's a heading element rather than a `div`.

Reply:
Thanks, I don't think there's an issue with the heading not being bolded as long as we're following the publishing components. I've pushed up a commit to change this!